forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -122,6 +132,70 @@ fn validate_and_parse_salts<'a>(
     Ok(contract_salt_map)
 }
 
+async fn deploy_chunked(
+    command: &cmd::Deploy,
+    compiled: &BuiltPackage,
+    salt: Salt,
+    signing_key: &SecretKey,
+    provider: &Provider,
+    pkg_name: &str,
+) -> anyhow::Result<(ContractId, Vec<ContractId>)> {
+    println_action_green("Splitting", &format!("{pkg_name} into chunks"));
```

Comment:
Rather than showing this message, we should update the `Confirming transactions` message to communicate this. It could look something like this:

`Confirming transactions [deploy big_contract + 5 contract chunks]`